src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,110 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String PASSWORD_WITH_LESS_THAN_SIX_CHARACTERS = "abc";
+    private static final String LOGIN_FOR_USER_IN_STORAGE = "uniqueLogin";
+    private static final int AGE_UNDER_EIGHTEEN = 15;

```

Comment:
or testing it is better to take marginally unacceptable input data, in this case 17 years (the same for invalid passwords)